sky/shell/platform/ios/FlutterViewController.mm

```diff
@@ -183,17 +187,27 @@ - (void)setupPlatformServiceProvider {
   mojo::ServiceProviderPtr viewServiceProvider;
   new sky::shell::ViewServiceProvider(mojo::GetProxy(&viewServiceProvider));
 
+  mojo::ServiceProviderPtr outgoingServiceProvider;
+
   sky::ServicesDataPtr services = sky::ServicesData::New();
   services->incoming_services = serviceProvider.Pass();
+  services->outgoing_services = mojo::GetProxy(&outgoingServiceProvider);
   services->view_services = viewServiceProvider.Pass();
   _engine->SetServices(services.Pass());
+
+  semantics::SemanticsServerPtr semantics_server;
+  mojo::ConnectToService(outgoingServiceProvider.get(), &semantics_server);
+  _semanticsServer = semantics_server.Pass();
 }
 
 #pragma mark - Loading the view
 
 - (void)loadView {
   FlutterView* surface = [[FlutterView alloc] init];
 
+  // Construct the listener implementation
+  new sky::shell::SemanticsListenerImpl(surface, _semanticsServer.Pass());
```

Comment:
Is there another place we create these?  Can loadView be called more than once?  Do we need to retain this, or presumably its retained by its connection?